cmd/todd-agent/main.go

```diff
@@ -51,6 +51,11 @@ func main() {
 
 	cfg := config.GetConfig(arg_config)
 
+	// Create optDir
+	if err := os.MkdirAll(cfg.LocalResources.OptDir, 0755); err != nil {
```

Comment:
What do you want done with the `make install`? `go install` should be run as the current user so packages end up in the correct correct location, it won't work correctly with sudo (unless `-E` is specified). Do you want to go back to using `go build` for each binary and place them in `/usr/bin` instead of `$GOPATH/bin`?